source/guide_prosody.rst

```diff
@@ -0,0 +1,404 @@
+.. highlight:: console
+.. author:: Arian Malek <https://fetziverse.de>
+.. author:: fapsi
+
+.. tag:: Instant Messaging
+.. tag:: Jabber
+.. tag:: XMPP
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/prosody.svg
+      :align: center
+
+#########
+Prosody
+#########
+
+.. tag_list::
+
+Prosody_ is a modern XMPP communication server. It aims to be easy to set up and configure, and efficient with system resources. Additionally, for developers it aims to be easy to extend and give a flexible system on which to rapidly develop added functionality, or prototype new protocols.
+
+XMPP is an open and free alternative to commercial messaging and chat providers. Set it up for your company, organisation, or just your family and friends. You are in control, and your communication is private to you. Supporting a wide range of client software for desktop and mobile platforms, you can chat using Prosody from any device.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Domains <web-domains>`
+  * :manual:`Firewall Ports <basics-ports>`
+  * :manual:`HTTPS <web-https>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+This guide is based on the initial `pull request`_ from fapsi_.
+
+License
+=======
+
+Prosody is open source software under the permissive MIT/X11 license.
+
+Prerequisites
+=============
+
+Web domain
+----------
+
+.. note:: To understand and setup the domains easier I use the following (recommended) subdomains:
+
+ * ``xmpp.isabell.uber.space``
+ * ``conference.isabell.uber.space``
+ * ``upload.isabell.uber.space``
+
```

Comment:
```suggestion

  Keep in mind that since you can't create DNS records for ``.uber.space`` domains, you'll need your own domain like ``isabell.org``.

```

Reply:
What do you think when I continue the whole guide with ``isabell.org`` instead of ``isabell.uber.space`` with the following note:

```suggestion
.. note:: Keep in mind that since you can't create DNS records for ``.uber.space`` domains, you'll need your own domain like ``isabell.org``. To understand and setup the domains easier I use the following (recommended) subdomains:

 * ``xmpp.isabell.org``
 * ``conference.isabell.org``
 * ``upload.isabell.uber.space``

Your URL ``xmpp.isabell.org`` and ``upload.isabell.org`` needs to be setup:

::

 [isabell@stardust ~]$ uberspace web domain list
 isabell.org
 xmpp.isabell.org
 upload.isabell.org
 [isabell@stardust ~]$
```

Reply:
since this cannot work with the default domain at all, sure!